src/box/iproto.cc

```diff
@@ -918,6 +918,8 @@ struct iproto_connection
 	 * connection.
 	 */
 	struct cmsg cancel_msg;
+	/** Set if connection is accepted in TX. */
+	bool is_connecting_finished;
```

Comment:
Better call `is_established`, I think.

Reply:
Done.

<details>
<summary>diff</summary>

```diff
diff --git a/src/box/iproto.cc b/src/box/iproto.cc
index fa1e82435..4cce543d9 100644
--- a/src/box/iproto.cc
+++ b/src/box/iproto.cc
@@ -919,7 +919,7 @@ struct iproto_connection
 	 */
 	struct cmsg cancel_msg;
 	/** Set if connection is accepted in TX. */
-	bool is_connecting_finished;
+	bool is_established;
 };
 
 /** Returns a string suitable for logging. */
@@ -1665,7 +1665,7 @@ iproto_connection_new(struct iproto_thread *iproto_thread)
 	con->session = NULL;
 	con->is_in_replication = false;
 	con->is_drop_pending = false;
-	con->is_connecting_finished = false;
+	con->is_established = false;
 	rlist_create(&con->in_stop_list);
 	rlist_create(&con->tx.inprogress);
 	rlist_add_entry(&iproto_thread->connections, con, in_connections);
@@ -3353,7 +3353,7 @@ net_send_greeting(struct cmsg *m)
 		iproto_msg_delete(msg);
 		return;
 	}
-	con->is_connecting_finished = true;
+	con->is_established = true;
 	con->wend = msg->wpos;
 	/*
 	 * Connect is synchronous, so no one could have been
@@ -3999,14 +3999,14 @@ iproto_do_cfg_f(struct cbus_call_msg *m)
 			 * after replication is breaked.
 			 *
 			 * Do not close connection that is not yet
-			 * finished connecting. Otherwise session
+			 * established. Otherwise session
 			 * on_connect/on_disconnect callbacks may be
 			 * executed in reverse order in case of yields
 			 * in on_connect callbacks.
 			 */
 			if (!con->is_in_replication &&
 			    con->state == IPROTO_CONNECTION_ALIVE &&
-			    con->is_connecting_finished)
+			    con->is_established)
 				iproto_connection_close(con);
 			/*
 			 * Do not wait deletion of connection that called

```

</details>